src/test/comment/postComments.test.js

```diff
@@ -0,0 +1,255 @@
+        })
+        .end((error, res) => {
+            expect(res).to.have.status(422);
+            expect(res.body.status).to.deep.equal('error');
+            expect(res.body.error[0].field).to.deep.equal('comment');
+            expect(res.body.error[0].message).to.deep.equal('Please enter comment.');
+            done();
+        });    
+    });
+
+    it('Should send a 422 error if comment is more than 350 characters', (done) => {
+        let text = 'ndwndndvhjjgefjgnefknbfekbnfhjfnflk;ldfnndfbndfbflbknldbnkbnld ndvmnbfbldldfnblnbjbldfbnlb';
+        // creates a long string
+        const longText = text+=text+=text+=text+=text+=text;
+        chai.request(app)
+        .post('/api/v1/comments')
+        .set('Authorization', myToken)
+        .send({
+            request_id: 3,
+            comment: longText
+        })
+        .end((error, res) => {
+            expect(res).to.have.status(422);
+            expect(res.body.status).to.deep.equal('error');
+            expect(res.body.error[0].field).to.deep.equal('comment');
+            expect(res.body.error[0].message).to.deep.equal('Too long. Enter a maximum of 350 characters');
+            done();
+        });    
+    });
+
+    it('Should send a 404 error if request does not exist', (done) => {
+        chai.request(app)
+        .post('/api/v1/comments')
+        .set('Authorization', myToken)
+        .send({
+            request_id: 10,
+            comment: 'Is there a way forward?'
+        })
+        .end((error, res) => {
+            expect(res).to.have.status(404);
+            expect(res.body.status).to.deep.equal('error');
+            expect(res.body.error).to.deep.equal('Request not found.');
+            done();
+        });
+    });
+
+    it('Should send a 403 error if request does not belong to user', (done) => {
+        chai.request(app)
+        .post('/api/v1/comments')
+        .set('Authorization', myToken)
+        .send({
+            request_id: 2,
+            comment: 'Is there a way forward?'
+        })
+        .end((error, res) => {
+            expect(res).to.have.status(403);
+            expect(res.body.status).to.deep.equal('error');
+            expect(res.body.error).to.deep.equal('You cannot add a comment to this request.');
+            done();
+        });
+    });
+
+    it('Should send a 403 error is user is neither the manager or owner of the request', (done) => {
+        chai.request(app)
+        .post('/api/v1/comments')
+        .set('Authorization', myToken)
+        .send({
+            request_id: 2,
+            comment: 'Is there a way forward?'
+        })
+        .end((error, res) => {
+            expect(res).to.have.status(403);
+            expect(res.body.status).to.deep.equal('error');
+            expect(res.body.error).to.deep.equal('You cannot add a comment to this request.');
+            done();
+        });
+    });
+
+    it('Should post comment if everything checks out', (done) => {
```

Comment:
Your implementation appears to have taken care of it but wouldn't you want to test the user's manager can post comments on the reqeust too?